docs/building/netcoreapp3.1-instructions.md

```diff
@@ -1,4 +1,4 @@
-ML.NET now builds for .NET Core 3.0 by default. However to run tests on .NET Core 3.0, you need to do a few manual steps.
+ML.NET now builds for .NET Core 3.1 by default. However to run tests on .NET Core 3.1, you need to do a few manual steps.
```

Comment:
Is this accurate? I think we build for netcoreapp21 when you run build.

Reply:
The src files always build on both netstandard2.0 and netcoreapp3.1. Meanwhile the tests only build on one target framework at a time: netcoreapp2.1, net461 or netcoreapp3.1 (Thanks @eerhardt !) I updated this document to reflect this.